src/internal/views.py

```diff
@@ -332,3 +333,100 @@ def has_permission(self):
                 self.request.user.has_perm('internal.delete_quote')
                 or self.request.user == self.get_object().author
         )
+
+
+class LoreListView(ListView):
+    template_name = 'internal/lore_wiki.html'
+    model = Lore
+    context_object_name = 'lore_topics'
+    ordering = 'title'
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['show_article'] = False
+        return context
+
+
+class LoreDetailView(DetailView):
+    template_name = 'internal/lore_wiki.html'
+    model = Lore
+    context_object_name = 'lore_article'
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['show_article'] = True
+        lore_topics = Lore.objects.all().order_by('title')
+        context['lore_topics'] = lore_topics
+        return context
```

Comment:
When I switch to using extra_context instead of overriding the get_context_data method, the list of lore articles added via the extra context does not update when a new lore article is added. So I am keeping this the way it is.

Reply:
Ah, true, I forgot that `'all_lore_topics'` is a queryset, which don't get refreshed between each request when assigned to a class field - as I mentioned in [this comment on another PR](https://github.com/MAKENTNU/web/pull/669#discussion_r1403798854) 😅

Context variables that are supposed to be constant, however, can preferably be listed in `extra_context`, I think, as it makes the view code ever so slightly clearer :) So:
```python
extra_context = {
    'show_article': True,
}
```